src/objectmap-utils/src/sdk.ts

```diff
@@ -124,17 +124,21 @@ export const createRecipe = (data: any) => {
   );
 };
 
-export const transformData = (data: any, sourceData: any) => {
+export const transformData = (data: any, sourceData: [{ [key: string]: any }]) => {
```

Comment:
@MikeFuster Can we use generics here? Should sourceData be `{[key: string]: any }[]`?